legend-engine-xt-mastery-pure/src/test/java/org/finos/legend/pure/code/core/mastery/Test_Pure_Mastery.java

```diff
@@ -0,0 +1,32 @@
+//  Copyright 2022 Goldman Sachs
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package org.finos.legend.pure.code.core.mastery;
+
+import junit.framework.TestSuite;
+import org.finos.legend.pure.code.core.compiled.test.PureTestBuilderHelper;
+import org.finos.legend.pure.m3.execution.test.TestCollection;
+import org.finos.legend.pure.runtime.java.compiled.execution.CompiledExecutionSupport;
+
+public class Test_Pure_Mastery
```

Comment:
The tests have been commented out. Is this deliberate ?

Reply:
It is intentional, I clarified with a comment.